$OpenBSD: patch-src_main_c,v 1.7 2010/11/08 15:04:15 dcoppa Exp $

set some sane defaults:
use mplayer's software volume control (see pkg/MESSAGE)
disable bar animation in full screen
use xscrnsaver controls over gnome-power-manager

--- src/main.c.orig	Tue Oct 19 18:33:40 2010
+++ src/main.c	Mon Nov  8 15:43:36 2010
@@ -689,9 +689,9 @@ int main(int argc, char *argv[])
     playlist_visible = FALSE;
     disable_fullscreen = FALSE;
     disable_framedrop = FALSE;
-    softvol = FALSE;
-    remember_softvol = FALSE;
-    volume_softvol = -1;
+    softvol = TRUE;
+    remember_softvol = TRUE;
+    volume_softvol = 50;
     volume_gain = 0;
     subtitlefont = NULL;
     subtitle_codepage = NULL;
@@ -705,7 +705,7 @@ int main(int argc, char *argv[])
     reallyverbose = 0;
     embedding_disabled = FALSE;
     disable_pause_on_click = FALSE;
-    disable_animation = FALSE;
+    disable_animation = TRUE;
     auto_hide_timeout = 3;
     use_mediakeys = TRUE;
     use_defaultpl = FALSE;
@@ -744,7 +744,7 @@ int main(int argc, char *argv[])
     start_second = 0;
     play_length = 0;
     save_loc = TRUE;
-    use_xscrnsaver = FALSE;
+    use_xscrnsaver = TRUE;
     screensaver_disabled = FALSE;
     update_control_flag = FALSE;
     gchar *filename;
